src/components/ErrorMessage.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './ErrorMessage.css';
+
+function ErrorMessage({ message }) {
+    if (!message) return null;
```

Comment:
Using ternary statements would be a better way to determine whether or not it renders